framework/packages/abstract-client/src/client.rs

```diff
@@ -122,6 +122,13 @@ impl<Chain: CwEnv> AbstractClient<Chain> {
         &self.abstr.ans_host
     }
 
+    /// Abstract Ibc Client contract API
+    ///
+    /// The Abstract Ibc Client contract allows users to create and use Interchain Abstract Accounts
+    pub fn ibc_client(&self) -> &IbcClient<Chain> {
```

Comment:
should we expose that on the abstract Client ? Feels too low level for me

Reply:
I think we should, don't think it should be hidden. We expose vc and ans as well!